### PR TITLE
chore(website): fix missing link to API docs in mobile view

### PR DESCRIPTION
**Pull request by aryanas159**
**Description**
Screenshot 2023-07-07 203530
Screenshot 2023-07-07 203546

Issue #1431 fixed, changes the mobile nav menu to include API docs link.

**Related issue(s)**

Fixes https://github.com/asyncapi/modelina/issues/1431

### CODE REVIEW

File: modelina-website/scripts/build-examples.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 const path = require('path');
 const {readdir, readFile, writeFile, mkdir, } = require('fs/promises');
 
+const refactorExampleName = (name) => {
+  const [first, ...rest] = name.split('-');
+  return first.charAt(0).toUpperCase() + first.slice(1) + " " + rest.join(" ")
+
+}
+
```
Comment (sambhavgupta0705) [inline modelina-website/scripts/build-examples.js:9]: what's the use of this code

Comment (aryanas159) [reply modelina-website/scripts/build-examples.js:9]: It's actually solving another issue #1417, I made a pull request before this for this change.

Comment (sambhavgupta0705) [reply modelina-website/scripts/build-examples.js:9]: well then make two different branches for both PR

Comment (jonaslagoni) [reply modelina-website/scripts/build-examples.js]: As @sambhavgupta0705 said, try not to depend two PRs with each other as much as possible 🙂 No problems this time, but if there was any issues in your last PR, this one would not be independently mergeable 😄

**Review by jonaslagoni — APPROVED** [review]

**Pull request merged**